### ISSUE TITLE
make 1.9ghz belize2 ps4 slim bzimage

### ISSUE THREAD

**Issue by zqpvr** [whitehax0r/ArchLinux-PS4v2#4]

**Comment by whitehax0r** [comment]
Have you try with the one on my github? this is the link: https://mega.nz/file/nyoFTZLA#13w00tZjVRWRGUXYWLgpmlyIJ0i4o__AWydgv1Nbq_8

**Comment by zqpvr** [comment]
Yes but it launches normally But wont autoboot. Typing resume-boot doesnt work
